### ISSUE TITLE
Wrong Cocoapods source url

### ISSUE THREAD

**Issue by CiriousJoker** [apache/cordova-ios#738]
`pod install` hangs at "Downloading specs".

This is because the Cocoapods team changed their hosting.

Check this out: [LINK](https://blog.cocoapods.org/CocoaPods-1.7.2/)

**TLRD:**
Change the first line of "Podfile" to "source 'https://cdn.cocoapods.org/" instead of the Github thing.

**Comment by timbru31** [comment]
Thanks for the pointer! Definitively something we should change. Do you happen to know if the new CDN works with older versions of CocoaPods, too? Otherwise, this would be a breaking change.

**Comment by jcesarmobile** [comment]
Which Cordova-iOS version are you using? Latest release included some fixes for cocoa pods 1.8.0+
https://github.com/apache/cordova-ios/commit/094779f6601a71eee489a1bae7a67cf91e8409c9
I created a project with that source and it worked without problems despite being a new computer, it downloaded everything.
What’s your cocoa pods version?

Also, I think the source comes from plugins, not from cordova itself.
But the cdn only works on 1.7.2+, which isn’t a requirement

**Comment by CiriousJoker** [comment]
I'm using cordova-ios v5.1.1. 

However, I'm not the only one with that issue, since I discovered the solution by following some other people's advice. Was hard to find though and I don't know where I found it..

**Comment by jcesarmobile** [comment]
I don't think it's hanging, it just takes a lot of time as it has to download all CocoaPods (about 10-20 minutes). The CDN is supposed to avoid the downloading, but as I said, the source comes from plugins.

But maybe cordova-ios should not write the source if it's the old one and the user is in CocoaPods 1.8.0+

**Comment by CiriousJoker** [comment]
To be honest, I have no idea how cocoapods works internally, I just noticed that it didn't work out of the box so i created the issue.

**Comment by jcesarmobile** [comment]
yeah, if the old url is used, CocoaPods downloads the whole CocoaPods repository into your computer, while the CDN url only download the required dependencies.
It only does it once, so next invocations are fast, but still, it wasted a lot of resources on the computer, that's why they created the CDN

**Comment by neuronas** [comment]
I tried to set new cdn from a virtualized mac OS Catalina commenting out the source line in platforms/ios/Podfile to increase the installation time performance to a one minute as mention on:
http://blog.cocoapods.org/CocoaPods-1.8.0-beta/

here are the relevant programs versions, and also I'm using cordova cli not cordova-ios but the same I think

% npm list -g --depth=0
├── cordova@9.0.0
└── npm@6.13.4

% xcodebuild -version
Xcode 11.3
Build version 11C29

% pod --version
1.8.4


pod install takes a minute but it didn't work when later execute cordoba build, 
also I tried removing Podfile.lock, remove all pods in Pods directory with the same result.
this is the error:

▸ Running script '[CP] Check Pods Manifest.lock'

❌  error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.

** ARCHIVE FAILED **

The following build commands failed:
	PhaseScriptExecution [CP]\ Check\ Pods\ Manifest.lock /Users/user/Library/Developer/Xcode/DerivedData/MyProjectName-bgxxntwvqlmwnocfgmzydttfdxuo/Build/Intermediates.noindex/ArchiveIntermediates/MyProjectName/IntermediateBuildFilesPath/MyProjectName.build/Release-iphoneos/MyProjectName.build/Script-76393A39D8E9B365915172E6.sh
(1 failure)
xcodebuild: Command failed with exit code 65
user@users-iMac MyProjectName % cd platforms/ios/ 



I ended up doing classic installation (pod install from platforms/ios) with default Podfile source, it takes almost 1 hour 

du -sh ~/.cocoapods
2.4G	/Users/user/.cocoapods
